django_simple_bulma/finders.py

```diff
@@ -71,10 +70,10 @@ def _get_bulma_css(self):
             )
 
         css_path = self.simple_bulma_path / "css" / "bulma.css"
-        with open(css_path, "w") as bulma_css:
+        with css_path.open("w") as bulma_css:
```

Comment:
```suggestion
        with open(css_path, "w") as bulma_css:
```
this is just personal preference.